README.md

```diff
@@ -368,5 +382,44 @@ const toast2 = showToast("number 2", configs);
 ```
 
 ### Styling the toast
-
 TODO
+
+## Security and Privacy Considerations
+See [TAG Security / Privacy Self Review](/security-privacy-self-review.md).
+
+## Considered Alternatives
+
+### Extending the Notification API
+Toasts are intended to be ephemeral, context-specific messages,
+and collecting them in a user's notifications tray doesn't reflect the spirit of the pattern.
+[kenchris]() put this well in [this comment]() on the TAG Design Review:
+
+> the difference is that this is a temporary notification / infobar, 
+> like you see for PWAs ("new version available, press reload to refresh") 
+> or like in GMail ("Chat is currently unavailable") etc. 
+> These are not things you want an actual system notification for... 
+> They really depend on the surrounding content/situation unlike regular notifications.
+
+Toasts also often contain some sort of action the user can take which affects the state of the page 
+(e.g. un-sending an email on Gmail).
+To provide this kind of behavior accessibly the element should live wholly in the page / DOM.
+This will have the added benefit of preventing pages in the background from using toasts to fight for user attention.
+
+### Extending the `<dialog>` element
+This approach was considered, but we decided to go a different route for a few reasons.
+First and foremost, we felt the accessibility considerations for a toast and a dialog are fundamentally opposed,
+as the dialog engages in the kind of flow-breaking behavior we want toasts to avoid.
+Additionally, the popular pattern of adding a call to action button on a toast with a baked-in time limit
+necessitates replacing tab-trapping with a less intrusive, easily restored alternative.
```

Comment:
Maybe we should soften this a bit by talking about how the exact differences between them are still being explored, and especially for action-button-containing toasts, the line might be a bit blurry. Then we could talk about how even if the line is blurry functionally, it's very different in terms of intent and semantics, similar to blockquote and q, or meter and progress.

Reply:
Done, hope it's more clear now.